galleries/examples/event_handling/viewlims.py

```diff
@@ -3,7 +3,7 @@
 Viewlims
 ========
 
-Creates two identical panels.  Zooming in on the right panel will show
+Creates two identical panels.  Zooming in on the left panel will show
```

Comment:
I'm not sure why this changed; the rectangle is still on the left, and zooming occurs on the right.